File: app/src/main/java/fmrsabino/moviesdb/ui/detail/DetailMvpView.java
package fmrsabino.moviesdb.ui.detail;

import fmrsabino.moviesdb.data.model.movie.Movie;
import fmrsabino.moviesdb.ui.base.MvpView;

public interface DetailMvpView extends MvpView {
    void getMovieDetails(Movie movie);
    void getCoverUrl(String url);
    void getPosterUrl(String url);
    void savedMovie(Movie movie);
}

File: app/src/main/java/fmrsabino/moviesdb/ui/detail/DetailMvpView.kt
package fmrsabino.moviesdb.ui.detail

import fmrsabino.moviesdb.data.model.movie.Movie
import fmrsabino.moviesdb.ui.base.MvpView

interface DetailMvpView : MvpView {
    fun getMovieDetails(movie: Movie)
    fun getCoverUrl(url: String)
    fun getPosterUrl(url: String)
    fun savedMovie(movie: Movie)
}
